x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/PhysicalPlanOptimizerTests.java

```diff
@@ -1132,6 +1132,45 @@ public void testPushMultipleBinaryLogicFilters() {
         assertThat(rq.to(), nullValue());
     }
 
+    public void testPushMultipleFunctions() {
```

Comment:
Please, add as a javadoc the expected physicalplan tree. I know some methods in this class do not have this, but I (or we) try to have this consistent throughout the class. Thanks.